ganttproject-tester/test/biz/ganttproject/impex/csv/GPSpreadsheetImportTest.java

```diff
@@ -272,4 +301,33 @@ public void testOutlineNumberComparator() {
     assertOrder("1.2", "1.10");
     assertOrder("2", "10");
   }
+
+  private Map<String, Task> doTestImportAssignments(byte[] data, SpreadsheetFormat format, TaskManagerBuilder builder,
```

Comment:
That's cool but my point was that not only test setup, but also test assertions were the same, and I suggested to write assertions once for both formats, rather than repeating them.